code/game/gamemodes/clock_cult/clock_proselytizer.dm

```diff
@@ -18,7 +18,7 @@
 
 /obj/item/clockwork/clockwork_proselytizer/scarab
 	name = "scarab proselytizer"
-	clockwork_desc = "A cogscarab's internal proselytizer. It can only be successfully used by a cogscarab and requires liquified Replicant Alloy to function."
```

Comment:
This capitalization is intentional; it's referring to the component alloy, which is always capitalized.